container/container.go

```diff
@@ -44,11 +44,21 @@ func (c *Container) Append(data []byte) {
 	c.compartments = append(c.compartments, data)
 }
 
+// PrependNumber prepends a number (varint encoded).
+func (c *Container) PrependNumber(n uint64) {
+	c.Prepend(varint.Pack64(n))
+}
+
```

Comment:
I'd rather name that `PrependInt64` to follow the naming convention of `PrependInt`. Otherwise `Number` is a bit ambiguous